benchmark/nixlbench/src/worker/nixl/nixl_worker.cpp

```diff
@@ -744,8 +744,8 @@ std::variant<double, int> xferBenchNixlWorker::transfer(size_t block_size,
 
     // Reduce skip by 10x for large block sizes
     if (block_size > LARGE_BLOCK_SIZE) {
-        skip /= LARGE_BLOCK_SIZE_ITER_FACTOR;
-        num_iter /= LARGE_BLOCK_SIZE_ITER_FACTOR;
+        skip /= xferBenchConfig::large_block_iter_factor;
```

Comment:
Why is the following missing here?
```
if (skip < MIN_WARMUP_ITERS) {
    skip = MIN_WARMUP_ITERS;
}
```

Number of skips and iterations in the transfer should match the poll